third_party/ibis/ibis_teradata/api.py

```diff
@@ -49,11 +49,13 @@ def connect(
         A Database username to connect with
     password : str
         Password for supplied username
+    logmech : str
+        Logmech flag to select with
     port : Optional[int]
         The database port to connect to (default. 1025)
     Returns
     -------
     TeradataClient
     """
 
-    return TeradataClient(host, user_name, password, port)
+    return TeradataClient(host, user_name, password, logmech, port)
```

Comment:
This won't work since the TeradataClient init method takes the parameters in a different order. I would put logmech after port and make it Optional. The [teradatasql.connect()](https://github.com/Teradata/python-driver#connection-parameters) documentation shows that "TD2" should be the default logmech

Reply:
Updated in the new commit.

Reply:
Logmech is still before port. When instantiating a TeradataClient class, this will fail.